server/execution/api/manvmap.py

```diff
@@ -0,0 +1,49 @@
+from random import random
+
+from flask import Blueprint
+from flask_jwt_extended import jwt_required
+
+from execution.utils import util
+
+api = Blueprint("api-manvmap", __name__)
+
+
+@api.get("/mapdata")
+@jwt_required()
+def get_map_data():
+    """ Returns mock map data. """
```

Comment:
Only partly correct. The patients and locations are in fact runtime entities. The only mockup here are the buildings. Probably mentionable :)

Reply:
Done!